diff --git a/node_modules/@ragetrade/vaults/contracts/base/RageERC4626.sol b/node_modules/@ragetrade/vaults/contracts/base/RageERC4626.sol
index 98454bd..69b4143 100644
--- a/node_modules/@ragetrade/vaults/contracts/base/RageERC4626.sol
+++ b/node_modules/@ragetrade/vaults/contracts/base/RageERC4626.sol
@@ -4,7 +4,7 @@ pragma solidity >=0.8.0;
 import { IERC20Metadata } from '@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol';
 import { SafeERC20 } from '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
 
-import { IBaseVault } from 'contracts/interfaces/IBaseVault.sol';
+import { IBaseVault } from '../interfaces/IBaseVault.sol';
 
 import { Logic } from '../libraries/Logic.sol';
 import { ERC4626Upgradeable } from '../utils/ERC4626Upgradeable.sol';
